src/styles/button.css

```diff
@@ -0,0 +1,29 @@
+/* card blocks elems with button parameters */
+.card__link-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 164px;
+  height: 38px;
+  line-height: 16px;
+
+  text-align: center;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+
+  background-color: var(--font-color_button);
+
+  border: 1px solid var(--font-color_button);
+  border-radius: 5px;
+
+  transition: 0.3s;
```

Comment:
specify the properties that can be changeable here, by default this transition will be used for all properties, and it can affect on website performance in bad way